px-data-grid-header-cell.html

```diff
@@ -267,7 +267,8 @@
           if (this._column && this._column._grid && this._column._grid._pxDataGrid) {
             const pxDataGrid = this._column._grid._pxDataGrid;
             const groupedByThis = pxDataGrid._groupByColumn == this._column;
-            return !groupedByThis && pxDataGrid.getVisibleColumns().length > 1;
+            const disableHide = this._column.disableHide;
+            return !disableHide && !groupedByThis && pxDataGrid.getVisibleColumns().length > 1;
```

Comment:
Might be a good idea to add another condition here checking whether the column dropdown is available.

I can't think of a valid use case for allowing users to hide columns irreversibly, and this means devs don't have to add `disable-hide` to every column just to resolve this issue. It also solves this problem for dynamically generated columns.

Reply:
I agree, will add this tomorrow as well as the doc updates